sonar-communityfsharp-plugin/pom.xml

```diff
@@ -44,7 +44,7 @@
         <groupId>org.sonarsource.sonarqube</groupId>
         <artifactId>sonar-plugin-api</artifactId>
         <!-- minimal version of SonarQube to support. -->
-        <version>7.9</version>
+        <version>7.9</version>        
```

Comment:
I don't like these trailing whitespaces, but GitHub offers an option to suppress them to view ;)

Reply:
i was opion the format option in the editor did clean that... apparently it did not